public/src/app.scss

```diff
@@ -3,17 +3,21 @@ $secondary: #606060;
 @import 'bootstrap/scss/bootstrap.scss';
 @import './_variables_&_mixins.scss';
 
+@import './shared/header';
```

Comment:
Doesn't this need to be `_header`?

Reply:
Webstorm says it's unnecessary but I'll update it so it's more standard.

Reply:
Normally it isn't necessary but for some reason on ubuntu it breaks without it